Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -7314,6 +7315,8 @@ def whois_request_get_response(socket, domain):
 class WhoisException(Exception):
     pass
 
+class WhoisExceptionWornning(Exception):
```

Comment:
Please change the name of this class, it has 2 issues:
1. Typo - the right word is "w**a**rning".
2. Exception should end with the word `exception` - e.g. `WhoisWarningException`.